source/core/src/main/com/csse3200/game/ui/DialogueBox/DialogueBox.java

```diff
@@ -358,6 +396,66 @@ public void handleBackwardButtonClick() {
         updateLabelPosition();
     }
 
+    /**
+     * Resizes and spaces the buttons based on the number of currently visible options.
+     */
+    public void resizeOptionButtons() {
```

Comment:
resize play button as well?

Reply:
Sure.

Reply:
This is now done please check you are happy with the new implementation. Regarding this since our team was not responsible for this do you expect us to develop tests?

Reply:
Nah I can do that, the play button doesn't still quite resize properly like the continue and back but I can have a look at that.

Reply:
I know exactly why. Can you provide the SetSize code you would like? I can do it the exact same but I was not asked to do this hence just fixed your positioning.

Reply:
Yeah I understand just thought while you were at it to make it the same as the others if it's easy. I'll approve without this tho and I'll fix it later if it's a bother (just don't feel like doing this course rn).